server/integration/infinispan/src/main/resources/org/jboss/as/clustering/infinispan/subsystem/LocalDescriptions.properties

```diff
@@ -134,6 +151,33 @@ infinispan.clustered-cache.replication-failures=Number of failed replications
 infinispan.clustered-cache.success-ratio=Successful replications as a ratio of total replications in numeric double format
 
 
+# clustered cache wide stats. See ClusterCacheStats
+infinispan.clustered-cache.clusterwide-number-of-locks-available=The clusterwide number of locks available in this cache.
+infinispan.clustered-cache.clusterwide-number-of-locks-held=The clusterwide number of locks currently in use by this cache.
+infinispan.clustered-cache.clusterwide-average-read-time=Clusterwide average time (in ms) for cache reads. Includes hits and misses. May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-average-remove-time=Clusterwide average time (in ms) for a cache removal. Includes hits and misses. May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-average-write-time=Clusterwide average time (in ms) for cache writes. May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-elapsed-time=Time (in secs) since first cache node started. May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-evictions=The clusterwide number of cache eviction operations. May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-hit-ratio=The clusterwide hit/miss ratio for the cache (hits/hits+misses). May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-hits=The clusterwide number of cache attribute hits. May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-misses=The clusterwide number of cache attribute misses. May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-number-of-entries=The clusterwide current number of entries in the cache. May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-read-write-ratio=The clusterwide read/write ratio of the cache ((hits+misses)/stores). May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-remove-hits=The clusterwide number of cache attribute remove hits. May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-remove-misses=The clusterwide number of cache attribute remove misses. May return null if the cache is not started.
+infinispan.clustered-cache.clusterwide-statistics-enabled=Whether statistics have been enabled for this cache.
```

Comment:
Hi @vblagoje I noticed  infinispan.clustered-cache.clusterwide-statistics-enabled is not used anywhere. Is that intention? Should I fix it? What do you think? 
I can pepare a fix tomorrow in case this is really a small bug. Sorry for nitpicking.